### PR TITLE
Show schema info

### PR DESCRIPTION
**Pull request by andrejtonev**
### Description

Please briefly explain the changes you made.

### Pull request type

Please check what kind of PR this is:

- [ ] Fix or improvement of an existing page
- [ ] New documentation page, release related

### Related PRs and issues

PR this doc page is related to: 
[2256](https://github.com/memgraph/memgraph/pull/2256)

### Checklist:

- [ ] Check all content with Grammarly
- [ ] Perform a self-review of my code
- [ ] Make corresponding changes to the rest of the documentation (consult with the DX team)
- [ ] The build passes locally
- [ ] My changes generate no new warnings or errors
- [ ] Add a corresponding label
- [ ] If release-related, add a product and version label
- [ ] If release-related, add release note on product PR

### CODE REVIEW

File: pages/querying/schema.mdx [hunk 1]
```diff
@@ -7,11 +7,171 @@ import { Callout } from 'nextra/components'
...
 
-Some queries are designed to retrieve information about the structure of the data within the database, for example, node labels and properties associated, as well as relationship types and properties.
+Schema queries are designed to retrieve information about the structure of the data within the database, for example, node labels and properties associated, as well as relationship types and properties.
 
-## Schema-related queries
+## Run-time schema tracking
 
-To use schema-associated queries, start Memgraph with the `--storage-enable-schema-metadata` [configuration flag](/configuration/configuration-settings#storage) set to `True`. The flag facilitates the utilization of a specialized cache designed to store specific metadata related to the database.   
+To use run-time schema tracking, start Memgraph with the `--schema-info-enabled` [configuration flag](/configuration/configuration-settings#storage) set to `True`.Once enabled, every change to the data layout will be tracked, and a derived schema will be automatically updated. Note that enabling this feature may result in a slight performance drop; for more details, refer to [Performance penalty](#performance-penalty).
```
Comment (hal-eisen-MG) [inline pages/querying/schema.mdx:14]: Missing a <space> between period and `Once`.
s/drop/decrease/g

File: pages/querying/schema.mdx [hunk 2]
```diff
@@ -7,11 +7,171 @@ import { Callout } from 'nextra/components'
...
+CREATE (:A:B);
+CREATE (:A:B{prop1:1});
+CREATE (:A:B{prop1:"str"});
+CREATE (:A:B{prop1:true});
+CREATE (:A:B{prop1:"str2", prop2:10.0});
+CREATE (:A:B{prop1:false,  prop2:20.0});
+CREATE (:A:B{prop1:false,  prop2:30.0});
+```
+
+The resulting nodes section would look like this:
```
Comment (hal-eisen-MG) [inline pages/querying/schema.mdx:102]: s/section/section of the schema/g

File: pages/querying/schema.mdx [hunk 3]
```diff
@@ -7,11 +7,171 @@ import { Callout } from 'nextra/components'
...
+* Enum::Name
+* Point2D
+* Point3D
+
+
+#### Node JSON example
+
+Consider the following queries:
+```
+CREATE (:A:B);
```
Comment (hal-eisen-MG) [inline pages/querying/schema.mdx:90]: Please include an example of a simple node with zero labels, and another simple node with a single label. After those two, then you can go into the more complex examples.

File: pages/querying/schema.mdx [hunk 4]
```diff
@@ -7,11 +7,252 @@ import { Callout } from 'nextra/components'
...
+}
+```
+
+#### Node JSON examples
+
+The following query adds a node without any edges, labels or properties.
+```
+CREATE ();
+```
+The nodes section of this simple schema is:
```
Comment (hal-eisen-MG) [inline pages/querying/schema.mdx:112]: Missing an apostrophe (possessive): should be `node's`

File: pages/querying/schema.mdx [hunk 5]
```diff
@@ -7,11 +7,252 @@ import { Callout } from 'nextra/components'
...
 
-Some queries are designed to retrieve information about the structure of the data within the database, for example, node labels and properties associated, as well as relationship types and properties.
+Schema queries are designed to retrieve information about the structure of the data within the database, for example, node labels and properties associated, as well as relationship types and properties.
 
-## Schema-related queries
+## Run-time schema tracking
 
-To use schema-associated queries, start Memgraph with the `--storage-enable-schema-metadata` [configuration flag](/configuration/configuration-settings#storage) set to `True`. The flag facilitates the utilization of a specialized cache designed to store specific metadata related to the database.   
+To use run-time schema tracking, start Memgraph with the `--schema-info-enabled` [configuration flag](/configuration/configuration-settings#storage) set to `True`. Once enabled, every change to the data layout will be tracked, and a derived schema will be automatically updated. Note that enabling this feature may result in a slight performance decrease; for more details, refer to [Performance penalty](#performance-penalty).
```
Comment (hal-eisen-MG) [inline pages/querying/schema.mdx:14]: I can't find the #performance-penalty anchor - did you mean #performance-tips?

File: pages/querying/schema.mdx [hunk 6]
```diff
@@ -7,11 +7,252 @@ import { Callout } from 'nextra/components'
...
+* `node_constraints` - list of all currently defined node constraints
+* `node_indexes` - list of all currently defined node indexes
+* `edge_indexes` - list of all currently defined edge indexes
+* `enums` - list of all currently defined enums and their values
+
+#### Exceptions
+
+Exceptions that might be thrown while executing `SHOW SCHEMA INFO` query:
+* `Show schema info is not supported for OnDisk.` - Please use InMemory Transactional or Analytical mode instead.
+* `Show schema info cannot be executed in multicommand transactions.` - Show schema must be executed in an [implicit (or auto-commit) transaction](#implicit-transactions).
```
Comment (hal-eisen-MG) [inline pages/querying/schema.mdx:40]: I can't find this #implicit-transactions anchor either

Comment (andrejtonev) [reply pages/querying/schema.mdx]: removed the link

**Review by hal-eisen-MG — APPROVED** [review]
LGTM

**Pull request merged**